### ISSUE TITLE
Error logged in InteractiveTestUtils.retrieveGetFeatureInfoRequest

### ISSUE THREAD

**Issue by rjmartell** [opengeospatial/ets-dgiwg#12]
An error is logged when the suite is run through the main web app. The root cause appears to be the following:

```
Caused by: java.lang.AssertionError: Could not find suitable layer for GetMap requests. expected object to not be null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.assertNotNull(Assert.java:423)
	at de.latlon.ets.wms13.core.util.request.WmsRequestBuilder.buildGetFeatureInfoRequestWithFormat(WmsRequestBuilder.java:128)
	at de.latlon.ets.wms13.core.util.request.WmsRequestBuilder.buildGetFeatureInfoRequest(WmsRequestBuilder.java:83)
...
```

### Test run arguments
`wms-uri` = https://tb12.cubewerx.com/a041/cubeserv?SERVICE=WMS&REQUEST=GetCapabilities&VERSION=1.3.0 
`wms-vector` = yes

**Comment by rjmartell** [comment]
When the suite was run via the REST API the following overall results were obtained:

skipped = 205
failed = 39
passed = 218

The configuration method `buildGetMapRequest()` often failed with the same error message: "Could not find suitable layer for GetMap request. expected object to not be null"

**Comment by dstenger** [comment]
The "queryable" attribute is set to "true" in the capabilities document.
The ets expects values "0|1" instead of "false|true". Because of that, all layers are treated as non queryable and there is no layer found which can be used for the tests.

**Comment by dstenger** [comment]
We created a fix for that problem and merged the pull request (#13) to master branch.
